packages/example-getting-started/package.json

```diff
@@ -14,12 +14,12 @@
     "build:dist": "lb-tsc es2017",
     "build:dist6": "lb-tsc es2015",
     "build:apidocs": "lb-apidocs",
-    "clean": "lb-clean loopback-getting-started*.tgz dist dist6 package api-docs",
+    "clean": "lb-clean *example-getting-started*.tgz dist dist6 package api-docs",
```

Comment:
Is the `*` before the word `example` necessary?

Reply:
Yes. Here is the full tarball name as produced by `npm pack`:

```
loopback-example-getting-started-1.0.1-alpha.0.tgz
```